### PR TITLE
kubeadm design doc

### PR DESCRIPTION
**Pull request by fabriziopandini**
As agreed with @luxas, this PR moves to the kubernetes web site the kubeadm design doc for release 1.9, formerly hosted in the kubeadm repository.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6663)

### CODE REVIEW

**Comment by zacharysarah** [comment]
- Changing base to `master` post-1.9 release
- Please resolve the merge conflicts

**Comment by fabriziopandini** [comment]
@zacharysarah rebase done! 👍

File: docs/reference/setup-tools/kubeadm/internal-design.md [hunk 1]
```diff
@@ -0,0 +1,581 @@
+---
+approvers:
+- mikedanese
+- luxas
+- jbeda
+title: Internal design
```
Comment (luxas) [inline docs/reference/setup-tools/kubeadm/internal-design.md]: On a second thought, "Implementation details" or "kubeadm under the hood" or similar could also make sense here

**Comment by luxas** [comment]
The netlify test needs to be fixed

**Comment by zacharysarah** [comment]
@luxas 👋 Sorry for the delay--yes, everything looks good to go. Merging now!

**Pull request merged**